app/controllers/cases/case/subcases/subcase/overview.js

```diff
@@ -46,7 +46,7 @@ export default class CasesCaseSubcasesSubcaseOverviewController extends Controll
     const governmentAreas = this.governmentAreas;
```

Comment:
It seems weird to me that `governmentAreas` is a tracked property here. This made sense when they came from the parent `case`, but now they are a part of the `subcase` itself. Why keep this property at all when they're in the model already?

Reply:
I assume this is done to handle ember-data relations being async. By using the tracked property and resolving the relation in the `afterModel`-hook of the route, the controller/template doesn't need to bother about the relation being async. In the template of the subcase route `this.governmentAreas` is passed as argument to the government-areas-panel component 
```hbs
      <Utils::GovernmentAreas::GovernmentAreasPanel
        @governmentAreas={{this.governmentAreas}}
        @allowEditing={{this.currentSession.isEditor}}
        @onSave={{this.saveGovernmentAreas}}
      />
```

But it's inconsistent with the way the data passing is done in the agendaitem template. There we do pass the async relation to the government-areas-panel component 
```hbs
  <Utils::GovernmentAreas::GovernmentAreasPanel
    @governmentAreas={{this.subcase.governmentAreas}}
    @allowEditing={{this.currentSession.isEditor}}
    @onSave={{this.saveGovernmentAreas}}
  />
```

We actually try to avoid passing promises as arguments to components, but it seems for this case the government-areas-panel component is aware that `@governmentAreas` might contain a promise since the value is [`yield`-ed in the constructor](https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/development/app/components/utils/government-areas/government-areas-panel.js#L42)

TL;DR You're right that the tracked property isn't necessary in the subcase route and the relation can be directly passed to the government-areas-panel component  as done in the agendaitem route.

Reply:
removed the tracked